$NetBSD: patch-aa,v 1.2 2010/09/23 21:58:13 sno Exp $

From p5-Math-GMP/patches/patch-aa:

Trick perl into emitting use of -lgmp.  Actual library
location will be handled via buildlink / wrapper.

--- Makefile.PL.orig	2011-02-26 14:34:32.000000000 +0000
+++ Makefile.PL
@@ -53,6 +53,6 @@ WriteMakefile(
                                 Math::BigInt => 1.993,
                                 XSLoader => 0.02,
                            },
-    'LIBS'              => ['-lgmp'],
+    'LIBS'		=> ["-L$ENV{PREFIX}/lib -lgmp"],
     'SIGN' => 1,
 );
